data/strings/types_strings.txt

```diff
@@ -21046,7 +20791,7 @@
     zh-Hant = 窗簾
 
   [type.shop.deli]
-    en = Delicatessen
+    en = Deli
```

Comment:
This is incorrect.  A delicatessen is sort of a place where you might buy higher-end food and a deli is where you buy sandwiches.   There is a tussle in OSM about the euro vs US meaning.   I would not wade into this and just leave this as it was.

Reply:
oke, leaving as it was :)